charts/ctlog/templates/createctconfig-job.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if not .Values.server.config.secret.enabled }}
```

Comment:
really sorry, looks like we have to keep the createctconfig and configmap.

createctconfig also creates the secret with the pub key, and store the treeID in the configmap.

even though we dont need the config data, there is no way to keep the others without this.
so i propose keeping everything, like you originally had in your PR. really sorry.
and filing an issue to make that part of the createctconfig configurable by flag to not generate the config.

Reply:
no worries, it's easy to switch back! created an issue (https://github.com/sigstore/scaffolding/issues/129) but specified we could add a flag to createctconfig to store the config in a secret, which would go well with this option being added to helm